man/systemd.network.xml

```diff
@@ -796,6 +796,33 @@
       </variablelist>
   </refsect1>
 
+    <refsect1>
+    <title>[AddressLabel] Section Options</title>
+
+      <para>An <literal>[AddressLabel]</literal> section accepts the
+      following keys. Specify several <literal>[AddressLabel]</literal>
+      sections to configure several addresse labels. IPv6 address labels are
+      used for address selection. See <ulink url="https://tools.ietf.org/html/rfc3484">RFC 3484</ulink>.
+      Precedence is managed by userspace, and only the label itself is stored in the kernel</para>
+
+      <variablelist class='network-directives'>
+        <varlistentry>
+          <term><varname>Label=</varname></term>
+          <listitem>
+            <para> The label for the prefix (an unsigned integer). 0xffffffff is reserved. This
+            key is mandatory. </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term><varname>Prefix=</varname></term>
+          <listitem>
+            <para>IPv6 prefix separated by a <literal>/</literal> character.
```

Comment:
hmm, i think this deserves a few more words, i.e. a brief explanation what an ipv6 prefix is (i.e. an address with a prefix length, separated by a slash)